integration-tests/support/pages/ComponentsPage.ts

```diff
@@ -103,8 +104,15 @@ export class ComponentPage extends AbstractWizardPage {
     cy.get(ComponentsPagePO.customBuildPipelineRadioBtn).click();
   }
 
-  checkStatusOnModal(statusLocator) {
-    cy.get(statusLocator, {timeout:150000}).should('be.visible');
+  verifyAndWaitforPRisSent() {
+    cy.contains(ComponentsPagePO.label, 'Merge pull request', { timeout: 150000 }).should('be.visible')
+    cy.contains('a', 'Merge in GitHub').should('be.visible')
+  }
+
+  verifyAndWaitforPRmerge() {
+    cy.contains(ComponentsPagePO.label, 'Custom', { timeout: 150000 }).should('be.visible')
```

Comment:
I see we are waiting for 2 and half of a minute for every PR related action (here and in verifyAndWaitforPRisSent). Do we need that long timeout? If so, what about having it as a constant?